spec/declaration.dd

```diff
@@ -467,14 +467,27 @@ $(GNAME VoidInitializer):
         used before it is set, undefined program behavior will result.
         )
 
-        $(UNDEFINED_BEHAVIOR If a void initialized variable's value is
-        used before it is set, the behavior is undefined.
+        $(IMPLEMENTATION_DEFINED If a void initialized variable's value is
```

Comment:
Please don't mix distinct changes like this in the same commit. Making the values of `void` initialized variables implementation defined has little to do with the topic of this PR. Allowing access to uninitialized memory has no place in a language that strives to be safe. If you disa "implementation defined" should be changed to "unspecified", as no implementation is ever going to define the exact bit pattern, unless it is zero, which would defeat the purpose of the feature.